t/stream-plugin/limit-conn.t

```diff
@@ -334,3 +334,22 @@ GET /test_concurrency
 --- error_log
 The value of the configured key is empty, use client IP instead
 --- stream_enable
+
+
+
+=== TEST 11: decrease
```

Comment:
Can I rewrite the test case? the test case looks not related to the reproducing step

Reply:
You can change it, thank you